ooni/nettest.py

```diff
@@ -591,11 +594,16 @@ def generateMeasurements(self):
                 test_instance._setUp()
                 test_instance.summary = self.summary
                 for method in test_methods:
+                    try:
+                        measurement = self.makeMeasurement(
+                            test_instance,
+                            method,
+                            test_input)
+                    except Exception:
+                        log.exception(failure.Failure())
```

Comment:
Gosh, you are right. It does forget the traceback when you pass the exception explicitly. I do see however that `twisted.python.log` internally will create an empty failure by default when you pass it `None`, so I guess if we are to go this way it's enough to just call `log.exception()`